### DISCUSSION TITLE
can't generate 3ddose file

### DISCUSSION THREAD

**Discussion by YapingQi** [nrc-cnrc/EGSnrc#869] [Questions] [answered]
I use a phase-space source to irradiate a 30cm^3 water phantom, the scoring voxel is 0.1cm^3, and history is 10^9. I use a parallel batch run, it shows finished, but there is no 3ddose file. I checked the log file and it stopped in BATCH No.2. Btw, there is a lock file showing there are still some batch didn't finish. 

When I enlarged the voxel size, which means smaller voxels, the output is all fine. 

Please anyone know what's wrong with that? Thank you in advance~

**Comment by rtownson** [comment, marked as answer]
Take a look at the .egslog files, probably found in the most recent `egsrun` directories - are there any errors?

If it is crashing with a smaller voxel size only, then you are probably running out of memory. Try parallelizing less, increasing the available memory, or using larger voxels.

**Reply by YapingQi** [reply]
Thank you for the reply. I found no error in .egslog file and the memory is big enough for the running. Less parallels work sometimes but not always. Btw, what's the function of this parameter : the thickness of the outer region for the water phantom ? I set it as 0 cm. 
[files.zip](https://github.com/nrc-cnrc/EGSnrc/files/8656463/files.zip)

**Reply by rtownson** [reply]
I think you're referring to the material that you can optionally include outside the phantom as padding, to ensure that your source starts within a defined material like air. I suggest using a non-uniform region outside the phantom that pads with air to cover your source location. This can help prevent crashes like the one you are experiencing - let me know whether or not it still occurs after making that change.
